airflow/bin/cli.py

```diff
@@ -1379,6 +1380,12 @@ class CLIFactory(object):
              "backfill-related DAG runs and start "
              "anew with fresh, running DAG runs"),
             "store_true"),
+        'rerun_failed_tasks': Arg(
+            ("--rerun_failed_tasks",),
+            ("if set, the backfill will auto-rerun all the failed tasks for the backfill"
+             " date range instead of throwing exceptions"),
```

Comment:
nit: alignment seems off
nitty nit: spaces are better as last char of previous line as opposed to first char of line

Reply:
thanks. updated.